docs/conf.py

```diff
@@ -192,3 +192,7 @@
     "numpy": ("https://docs.scipy.org/doc/numpy/", None),
     "scipy": ("https://docs.scipy.org/doc/scipy/reference/", None),
 }
+
+doctest_global_setup = """
+import uarray as ua
```

Comment:
comment mostly unrelated to this PR: I think it's actually better to just say `import uarray`. Library devs won't use uarray all the time, so explicit is better than saving 3 chars.

Reply:
The protocols are also all `__ua_*__`... And changing those would mean changing the API (again). I'm doing this for uniformity...